src/openzaak/config/tests/setup_configuraiton/test_selectielijst_configuration.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2024 Dimpact
+from pathlib import Path
+
+from django.test import TestCase
+
+from django_setup_configuration.test_utils import execute_single_step
+
+from openzaak.config.setup_configuration.steps import SelectielijstAPIConfigurationStep
+from openzaak.selectielijst.models import ReferentieLijstConfig
+
+CONFIG_YAML = Path(__file__).parent / "files/selectielijst_api_config.yml"
+
+
+class SelectielijstConfigurationTests(TestCase):
+    def test_configure(self):
+
+        execute_single_step(
+            SelectielijstAPIConfigurationStep, yaml_source=str(CONFIG_YAML)
+        )
+
+        config = ReferentieLijstConfig.get_solo()
+        self.assertEqual(
+            config.service.api_root, "https://selectielijst.example.com/api/v1/"
+        )
```

Comment:
Could you add assertions for all fields on the config here to verify that they're correctly set here? It might also be good to add another test in which you explicitly specify `allowed_years` and `default_year` in the yaml file to verify that that works as well. You can add the Service via a factory in `setUpTestData`, since that should be created by the step from `zgw-consumers`